app/assets/stylesheets/application.css.scss

```diff
@@ -16,9 +16,38 @@
  *= require_self
 
  */
+$neutral: #d9d9d9;
 
 @import "font-awesome";
 
 legend a {
   font-weight: lighter;
 }
+
+.side-column {
+  font-weight: lighter;
+  text-align: center;
+  ul {
+    width: 70%;
+    margin: auto;
+  };
+  li {
+    list-style: none;
+    // border: 1px solid $neutral;
+    box-shadow: 0 1px 0 #ccc,
+               0 2px 0 #c9c9c9,
+               0 3px 0 #bbb,
+               0 4px 0 #b9b9b9,
```

Comment:
Color literals like `#b9b9b9` should only be used in variable declarations; they should be referred to via variable everywhere else.